src/main/java/com/cleanroommc/groovyscript/compat/mods/multiblocked/Multiblocked.java

```diff
@@ -0,0 +1,23 @@
+package com.cleanroommc.groovyscript.compat.mods.multiblocked;
+
+import com.cleanroommc.groovyscript.GroovyScript;
+import com.cleanroommc.groovyscript.compat.mods.AdvancedGroovyPropertyContainer;
+import com.cleanroommc.multiblocked.api.recipe.RecipeMap;
+
+public class Multiblocked extends AdvancedGroovyPropertyContainer {
+
+    @Override
+    public void onCompatLoaded() {
```

Comment:
theres already a way to do this, its the `initialize(GroovyContainer<?>)` method. no need to have an `AdvancedGroovyPropertyContainer` for it

Reply:
I also did not like this crutch, I will see how other mods did it. But I did find that `onCompatLoaded`'s body is empty for InternalModContainers...